pdal/plang/Environment.cpp

```diff
@@ -254,68 +258,59 @@ PyObject *fromMetadata(MetadataNode m)
         return o;
     };
 
-    PyDict_SetItemString(data, "name", getString(name));
-    PyDict_SetItemString(data, "value", getString(value));
-    PyDict_SetItemString(data, "type", getString(value));
-    PyDict_SetItemString(data, "description", getString(description));
+    PyObject *data = PyDict_New();
```

Comment:
PyDict_New could return NULL according to the docs.

Reply:
I'm not that concerned, I just never noticed it before and I probably wrote the original code here.